build-system/tasks/bundle-size/index.js

```diff
@@ -207,6 +207,9 @@ async function reportBundleSize() {
   }
 }
 
+/**
+ * @return {Promise<void>}
+ */
```

Comment:
Re: this change, @samouri is gonna say _"I told you so"_. And I will sheepishly look away and whistle a tune.

Reply:
ooh, the sweet sweet taste of being right on the internet